evalml/data_checks/target_distribution_data_check.py

```diff
@@ -142,3 +126,26 @@ def validate(self, X, y):
             )
 
         return results
+
+
+def _detect_log_distribution_helper(y):
```

Comment:
Moving mathy-implementation chunk into helper method to make `validate` easier to read!